src/platform/esp32/main-esp32.cpp

```diff
@@ -107,6 +107,11 @@ void esp32Setup()
     randomSeed(seed);
     */
 
+#ifdef POWER_FULL
+    pinMode(POWER_FULL, INPUT);
+    pinMode(7, INPUT);
+#endif
```

Comment:
No idea what 7 does, not in the schematic.

Reply:
Lucky GPIO number 7